packages/ui/components/Avatars/index.tsx

```diff
@@ -12,21 +13,20 @@ export const PersonAvatar: React.FC<PersonAvatarProps> = ({ personId }) => {
 };
 
 interface ProfileAvatar {
-  userId: string;
   imageUrl?: string | null;
+  email: string;
 }
 
-export const ProfileAvatar: React.FC<ProfileAvatar> = ({ userId, imageUrl }) => {
-  if (imageUrl) {
-    return (
-      <Image
-        src={imageUrl}
-        width="40"
-        height="40"
-        className="h-10 w-10 rounded-full object-cover"
-        alt="Avatar placeholder"
-      />
-    );
-  }
-  return <Avatar size={40} name={userId} variant="bauhaus" colors={colors} />;
+export const ProfileAvatar: React.FC<ProfileAvatar> = ({ imageUrl, email }) => {
+  const imageSrc = imageUrl || `https://www.gravatar.com/avatar/${sha256(email)}?d=retro`;
+
+  return (
+    <Image
+      src={imageSrc}
+      width="40"
+      height="40"
+      className="h-10 w-10 rounded-full object-cover"
+      alt="Avatar placeholder"
+    />
+  );
```

Comment:
Did you remove the `<Avatar />` component? I think it's still a good idea to return the boring-avatar if neither an image is uploaded by the user nor a gravatar is present since I think this is the most common case.

Reply:
Yes, because Gravatar will return a default image (e.g., d=retro), the <Avatar /> won't be needed.

Reply:
I would prefer using our boring avatars as default since I'm not a fan of the default gravatar images ;-)

Reply:
@mattinannt Gravatar will always return a default image if no avatar is found. Unfortunately, we cannot fall back to the boring-avatar React component. However, we can replace Gravatar's default image using the boring-avatar API, like this: https://www.gravatar.com/avatar/${hashedEmail}?d={CUSTOM_DOMAIN}?name={NAME}&variant={VARIANT}&size={SIZE}&colors={COLORS}&square.

Are you planning to use this service?